opentelemetry-sdk/src/metrics/pipeline.rs

```diff
@@ -403,16 +413,16 @@ where
 
             otel_debug!(
                 name : "Metrics.InstrumentCreated",
-                instrument_name = stream.name.as_ref(),
+                instrument_name = stream.name.clone().unwrap_or_default().as_ref(),
```

Comment:
We are being forced to introduce a lot of `unwrap_or_default()` method calls even though we know that the stream provided to `cached_aggregator` method would always have some name, unit, and description.

We should ideally create another internal struct that would have the same fields as `Stream` but they wouldn't be optional. Using such a struct instead of `Stream` would allow to get rid of these `unwrap_or_default()` calls which have weakened the contract for `cached_aggregator` method.

Reply:
good point! we can modify this separately, as we use the unwrap_default for buckets, attribute_keys too.